services/minespace-web/src/components/modalContent/config.js

```diff
@@ -3,11 +3,11 @@ import AddVarianceModal from "@/components/modalContent/variances/AddVarianceMod
 import ViewVarianceModal from "@/components/modalContent/variances/ViewVarianceModal";
 import EditVarianceModal from "@/components/modalContent/variances/EditVarianceModal";
 
-// This file is anticipated to have multiple exports
-// eslint-disable-next-line import/prefer-default-export
 export const modalConfig = {
   ADD_REPORT: AddReportModal,
-  ADD_VARIANCE: AddVarianceModal,
+  CREATE_VARIANCE: AddVarianceModal,
```

Comment:
We're inconsistent on nomenclature here. 
  **ADD**_REPORT: **Add**ReportModal,
  **CREATE**_VARIANCE: **Add**VarianceModal,

Reply:
This was a mishap on my part for forgetting to rename the other part too. I like to stick with CRUD action verbs for things like this. "Add" reads weird in my brain for "Create" actions. For example, do we "Subtract" something when we "Delete" it? Just a minor, very, very, important detail, obviously ;)